lib/expandable_page_view.dart

```diff
@@ -73,12 +74,17 @@ class _ExpandablePageViewState extends State<ExpandablePageView> {
     super.initState();
     _pageController = widget.controller ?? PageController();
     _pageController.addListener(_updatePage);
+    if (widget.controller == null) {
+      _shouldDisposePageController = true;
+    }
```

Comment:
For better readability i would change it to: 
```suggestion
    _shouldDisposePageController = widget.controller == null;
```